src/components/molecules/AdminVenueCard/AdminVenueCard.tsx

```diff
@@ -6,6 +6,7 @@ import { AnyVenue } from "types/venues";
 import { DEFAULT_VENUE_LOGO } from "settings";
 
 import { WithId } from "utils/id";
+import { adminV3SettigsUrl, adminv3VenueUrl } from "utils/url";
```

Comment:
Slight naming inconsistency with `*V3*` vs `*v4*`, so maybe just drop the letter V so we don't always think about proper casing of single letter words. Just `admin3` and `Admin3` are clear enough I think.

Reply:
This is a topic I want to talk about and I will bring it up in slack. Currently from product/design perspective this is 'Admin v3' but on our side we are actually replacing Admin v2 with Admin v3 and there will be no trace of Admin v2 in our codebase. It will be very miss-leading to have Admin and Admin v3 logic in our codebase. I suggest we come up with a better temporary name for these new screens until it's all implemented and becomes the only final Admin.
I'd really like everyone's input on this topic. @sparkletown/engineering

Reply:
@denisdimitrov can you please link the slack thread to this issue for continuity

Reply:
Fwiw, I’m 👎🏻 on dropping the v and just having a numeric suffix

Reply:
NG has been used for naming: https://github.com/sparkletown/sparkle/pull/1511/commits/2afccca403af90d38dca5a81a91041709b2afac3

Reply:
Leaving slack link for continuity: https://doesitsparkle.slack.com/archives/C01H9D1P1PE/p1623149690277100